recipes/ucsc_tools/meta.yaml

```diff
@@ -1,19 +1,19 @@
 package:
   name: ucsc_tools
-  version: "332"
+  version: "357"
 
 build:
   number: 1
```

Comment:
@natefoo the build number should be reset to 0.

Reply:
Whoops, thanks